### ISSUE TITLE
Question on min_val in split_candidate_manager

### ISSUE THREAD

**Issue by erchiw** [Samuel-Maddock/federated-boosted-dp-trees#3]
Hello Samuel,

I have a  question in /models/gbdt/components/split_candidate_manager.py

At line 112, why flip the sign if min_val > 0? I thought the sketch should be operated on the interval [min_val, max_val]? 

Thank you!

**Comment by Samuel-Maddock** [comment]
Hi Erchi, thanks for this issue!

> At line 112, why flip the sign if min_val > 0? I thought the sketch should be operated on the interval [min_val, max_val]?

Great point, this is indeed a bug leftover from old code that experimented with giving PrivateGBDT incorrect feature ranges to study how the performance behaves.

 I have verified that this has a minimal effect on accuracy for the adult and Higgs dataset. I have fixed this as of 37f7117a3b2a3b5cd729ff6d8c3bbd4d121a5818.
